src/css/map-view.css

```diff
@@ -561,43 +562,68 @@ represents 1 unit of the given distance measurement. */
         background-color: var(--map-col-content-bkg, var(--map-col-bkg));
         color: var(--map-col-text-body, var(--map-col-text));
         font-family: var(--map-body-font);
-        border: 1px solid;
-        border-color: var(--map-col-border-muted);
+        border: none;
         box-shadow: unset;
 
         ::placeholder {
           color: var(--map-col-text-muted);
         }
-
-        &.search-input__error-input {
-          border-color: var(--map-col-input-error-border);
-          background-color: var(--map-col-input-error-bkg);
-        }
       }
     }
+
+    &:focus-within {
+      border-color: var(--map-col-border-highlight-search-box, var(--map-col-blue));
+    }
+
+    &.search-input__error-input,
+    &.search-input__error-input input[type=text].search-input__input {
```

Comment:
is this selector repetitive?

Reply:
I removed absolute positioning, so the input field and the parent div both need to be styled when there is an error. I'll move the styles to just change background of the parent div, and the input element will have no background.